Testing/SystemTests/tests/framework/MuonMaxEntTest.py

```diff
@@ -7,14 +7,15 @@
 # pylint: disable=no-init,attribute-defined-outside-init
 import systemtesting
 from math import pi
-from mantid.simpleapi import *
+from mantid.api import AnalysisDataService
+from mantid.simpleapi import CreateEmptyTableWorkspace, GroupWorkspaces, Load, MuonMaxent
 
 
 class MuonMaxEntTest(systemtesting.MantidSystemTest):
     """Tests the MaxEnt algorithm on a MUSR workspace"""
 
     def fixPhasesTest(self, phases0):
-        MuonMaxEnt(
```

Comment:
Same with this... was this working before?

Reply:
I believe this `fixPhasesTest` is never run because only `runTest` is run in a system test. I decided it was out of scope for the PR to try fix it